parquet/src/main/java/org/apache/iceberg/parquet/ParquetUtil.java

```diff
@@ -297,6 +297,10 @@ public static boolean hasNonDictionaryPages(ColumnChunkMetaData meta) {
     }
   }
 
+  public static boolean hasNonBloomFilterPages(ColumnChunkMetaData meta) {
+    return meta.getBloomFilterOffset() == -1;
```

Comment:
As the `getBloomFilterOffset` is marked as `Private` method,  I would prefer not to use it in the production files , in case of breaking the compatibility when upgrading the parquet version in future. 

Is there any other approach to check whether it generate bloom filter binary or not ?

Reply:
I did some search and didn't find another approach. `ColumnChunkMetaData#getBloomFilterOffset` is also used when reading bloom filter to judge if bloom filter is enabled, see [ParquetFileReader#readBloomFilter](https://github.com/apache/parquet-mr/blob/db75a6815f2ba1d1ee89d1a90aeb296f1f3a8f20/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java#L1182).